os/arch/arm/src/Makefile

```diff
@@ -1,6 +1,6 @@
 ###########################################################################
 #
-# Copyright 2016 Samsung Electronics All Rights Reserved.
+# Copyright 2022 Samsung Electronics All Rights Reserved.
```

Comment:
Please revert this change. For this file, previous is correct.

Reply:
Yes Mr Chang I will revert to 2016.